app/views/analytics/_google_tag_manager.haml

```diff
@@ -6,11 +6,12 @@
       dl.push({"event": "identify", "id": "mp-"+marketplaceId+"-admin" });
     }
 
-    dl.push({
-      event: 'feature flags',
-      featureFlags: #{feature_flags.to_json.html_safe}
-    });
-
+    if (#{feature_flags.present?}){
```

Comment:
`feature_flags` seems to be undefined, so I guess this will break? Maybe worth testing again.